### PR TITLE
[terra-dev-site-v7] Update browserlist

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

This PR updates browserlist config and trims what get packaged down to a minimum.

Closes #

### Deployment Link

https://terra-dev-site-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +63,11 @@
       "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
     }
   },
+  "files": [
+    "lib",
+    "src",
+    "translations"
+  ],
```
Comment (emilyrohrbough) [inline package.json:70]: yess!!

**Review by emilyrohrbough — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -66,6 +63,11 @@
       "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
     }
   },
+  "files": [
```
Comment (StephenEsser) [inline package.json:66]: Can the .npmignore file be removed with these changes or is it still necessary?

**Review by StephenEsser — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (mjhenkes) [reply package.json:66]: That makes sense. I'll remove it and double check by running the pack command.

Comment (mjhenkes) [reply package.json:66]: It doesn't appear to be needed

**Pull request merged**